### ISSUE TITLE
os.pread not available in Python2

### ISSUE THREAD

**Issue by stefantalpalaru** [vstinner/perf#39]
Error when running "pyperf system show":
```
  File "/usr/lib64/python2.7/site-packages/perf/_system.py", line 146, in read_msr
    data = os.pread(fd, size, reg_num)
AttributeError: 'module' object has no attribute 'pread'
```

**Comment by vstinner** [comment]
Hum, maybe you should use Python 3 :-)

**Comment by stefantalpalaru** [comment]
> Hum, maybe you should use Python 3 :-)

I'm as inclined to use use Python3 instead of Python2 as you are to use Perl6 instead of Perl5.

Seriously, though, I'm packaging your software and it advertises Python2 support which is pretty important for comparing benchmarks between languages and implementations using the "performance" module.

**Comment by vstinner** [comment]
Can you try to write a patch to use seek()+read() when pread() on Python 2?

**Comment by vstinner** [comment]
Fixed by commits e82849af0e59ec792ea2320840f8420e3328422a and 5696ed64bfe69a61f5c69e17656777e0b03f44ff.
